src/bot/middleware/only-with-text.middleware.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * @description
+ * Skip messages without text
+ * */
+function onlyWithText(ctx, next) {
+  /**
+   * Skip messages before bot became admin
+   * */
+  if ((ctx.msg?.date || 0) * 1000 < +ctx.session.botAdminDate) {
```

Comment:
It's not transparent to use this in the `onlyWithText`. Could we make a separate middleware for this?